shared/product.go

```diff
@@ -69,10 +69,11 @@ func ManageService(product, action, nodeType string, ips []string) (string, erro
 
 func SystemCtlCmd(product, action, nodeType string) (string, error) {
 	systemctlCmdMap := map[string]string{
-		"stop":    "sudo systemctl --no-block stop",
-		"start":   "sudo systemctl --no-block start",
-		"restart": "sudo systemctl --no-block restart",
-		"status":  "sudo systemctl --no-block status",
+		"stop":            "sudo systemctl --no-block stop",
```

Comment:
Can add a var `systemCtlPrefix := "sudo systemctl --no-block "` and append the action

Reply:
dont see a need for that, since the map is already validating the actions....could u elaborate?

Reply:
and tbh the way it is it more clean, readable for me  like seeing the whole stuff 

```
	"stop":            "sudo systemctl --no-block stop",
		"start":           "sudo systemctl --no-block start",
		"restart":         "sudo systemctl --no-block restart",
		"status":          "sudo systemctl --no-block status",
		"restart-systemd": "sudo systemctl  --no-block restart systemd-sysctl",
```

Reply:
I was thinking of reducing the duplication of `sudo systemctl --no-block`

Reply:
got it...for me the value of having like that more readable here,  over exceeds changing to just not duplicate that keyword